recipes/autoconf/all/test_package/conanfile.py

```diff
@@ -1,47 +1,53 @@
-from conans import AutoToolsBuildEnvironment, ConanFile, tools
-from conan.tools.microsoft import is_msvc
-import contextlib
-import os
 import shutil
+from os import path
 
-required_conan_version = ">=1.45.0"
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.gnu import Autotools
+from conan.tools.microsoft import unix_path
+
+required_conan_version = ">=1.50.0"
```

Comment:
```suggestion
required_conan_version = ">=1.51.3"
```

Not necessary, just suggested since it is easy to commit.